bin/oneoff/backfill_data/channel_tokens_storage_app_id.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+
+# This script populates the storage_apps_id column of the channel_tokens table.
+
+require_relative '../../../dashboard/config/environment'
+require_relative '../../../shared/middleware/helpers/storage_id'
+
+ChannelToken.find_each do |channel_token|
+  storage_app_id, _channel_id = storage_decrypt_channel_id(channel_token.channel)
+  channel_token.storage_app_id = storage_app_id
+  channel_token.save!
+end
```

Comment:
ugh, now I'm sad I agreed to let you call this storage_app_id. these params are backwards, it should be `channel_token.storage_app_id = channel_id`. The `storage_id` is an id associated with the owner of the channel, and a channel is a thing in the storage_apps table.